benchmarks/dynamo/ci_expected_accuracy/inference_huggingface.csv

```diff
@@ -0,0 +1,43 @@
+dev,name,batch_size,accuracy,calls_captured,unique_graphs,graph_breaks,unique_graph_breaks
```

Comment:
It's not obvious to me that we actually want all these columns here? calls_captured in particular feels likely to wobble, in ways that we ought not to care about?

Reply:
i thought about this, and it seems easier to check in the exact csv that is easy to produce using our current flows.

and to your point about wobble, we just don't check the other fields- we check "graph_breaks"

Reply:
i think the cleanest would be 
1) the msg also tells you "run this command: pytorch/benchmarks/dynamo/update_expected.py --pr {PRNUM}"
2) there is some clever mapping so the script can download the right .zips, and extract the csvs (2 csv per zip currently)
3) it can also spit out newly stripped CSVs that only contain the cared-about columns
4) and 'git add' the changes so you don't have to 

But i have to figure out (2), not sure if it is trivial or we have to do something to make it possible.  CC @ZainRizvi any help here?  (best practices for (a) how to interact with artifact files from a script (b) any stable 'ENV' inside test jobs that i can use to build a URL to download artifacts etc

Reply:
For (2), you need an action after the test runs finish (see https://github.com/pytorch/pytorch/pull/95675 as an example). I am ok with doing this as a followup PR.